delphi-frontend/src/main/java/org/sonar/plugins/communitydelphi/api/reporting/QuickFix.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Sonar Delphi Plugin
+ * Copyright (C) 2024 Integrated Application Development
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonar.plugins.communitydelphi.api.reporting;
+
+import com.google.errorprone.annotations.FormatMethod;
+import com.google.errorprone.annotations.FormatString;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class QuickFix {
```

Comment:
We can, but there won't really be any benefit to it. I guess `getEdits` won't need to be visible, but since that's just a getter I feel like it doesn't matter all that much.

FWIW, SonarJava does not use an interface and impl class for this.

Reply:
Also - as I mention in my other comment, interfaces can't have static routines. If we had an interface and impl situation we'd have to store the static routines on another API-accessible class.

Reply:
Ugh. Wait, I just looked it up. Looks like you can technically have static routines on interfaces. Sounds like a bad idea still, IMO.

Reply:
SonarJava doesn't expose quick-fixes through their API at all, which is a key difference here. That's probably why they didn't have an interface layer.